Header responsive, creacion de dir pages y cambios en dir public.

gianpetrii opened the pull request:
https://alkemy-labs.atlassian.net/browse/OT242-21
EL header se hizo responsive, a evaluar si sacar los botones como componentes e importarlos.
Creacion de directorio 'pages' y llamado desde App.js en español ya que se ve en la url.
Cambio de titulo e imagen de pagina y eliminacion de contenido que no se utilizara.

[evidencia]
(no estoy pudiendo subir evidencia)

gianpetrii committed:
Header responsive, creacion de dir pages y cambios en dir public.

EL header se hizo responsive, a evaluar si sacar los botones como componentes e importarlos.
Creacion de directorio 'pages' y llamado desde App.js en español ya que se ve en la url.
Cambio de titulo e imagen de pagina y eliminacion de contenido que no se utilizara.

gianpetrii committed:
Agrego update de package y package-lock que olvide en commit previo

GauntStrelok commented on src/components/Header/Header.jsx:
```diff
@@ -0,0 +1,63 @@
+                    : <FaTimes className="hamburger" onClick={HandleMobileMenu}/>
+                }
+
+                <ul className={showMobileMenu}>
```
No veo que el menu de navegacion sea un array

gianpetrii committed:
tems del nav dentro de array y manejo de token

Meti los items del nav dentro de un array con el path de cada uno y un
key para identificarlos. Aclaro que deje los botones afuera ya que tengo
que ver con los chicos si usamos un componente "boton" y lo importamos
al header.
Traigo el token del localStorage y renderizo condicionalmente login o
logout segun corresponda

gianpetrii committed:
Elimino el package-lock como dijo Pablo :)

GauntStrelok commented:
Hacer pull de main para arreglar conflictos

gianpetrii committed:
Arreglo de conflicto en App.js y package

gianpetrii committed:
Borre App.js para evitar conflictos ya que no corresponde a la tarea

GauntStrelok commented:
Hacer pull de main para arreglar conflictos, avisar por discord cuando esten arreglados, por el canal pertinente
